sekoiaio/info.json

```
@@ -17,15 +20,18 @@
                 "required": true,
                 "editable": true,
                 "visible": true,
-                "type": "text"
+                "type": "password",
+                "description": "The API key to authenticate requests"
             },
             {
                 "title": "Verify certificate",
                 "name": "verify_certificate",
                 "required": false,
                 "editable": true,
                 "visible": true,
-                "type": "boolean"
+                "type": "checkbox",
```

same thing here, the type should be boolean otherwise it doesn't work
```suggestion
                "type": "boolean",
```